core/src/main/java/org/apache/calcite/rex/RexUtil.java

```diff
@@ -1683,7 +1683,7 @@ public static boolean isLosslessCast(RexNode node) {
    *
    * @param source source type
    * @param target target type
-   * @return true iff the conversion is a loss-less cast
+   * @return true if the conversion is a loss-less cast
```

Comment:
`iff` mean `if and only if`

Reply:
I know, but this function does not "true" if and only if, it returns it only "if", because it is an aproximation